common/test/acceptance/tests/studio/test_studio_settings.py

```diff
@@ -290,3 +290,10 @@ def set_wrong_inputs_to_fields(self):
                 "Course Announcement Date": '"string"',
             }
         )
+
+    def test_only_expected_fields_are_displayed(self):
+        expected_fields = self.advanced_settings.get_expected_settings_names()
```

Comment:
A couple of small requests--

1) Include doc describing what the test point does. Our best practice is BDD description, though this test isn't following that convention. For example, https://github.com/edx/edx-platform/blob/master/common/test/acceptance/tests/discussion/test_cohort_management.py#L81

2) Test that the length of expected_fields == length of displayed_fields. Otherwise, some fields that are expected to be shown may not actually be shown.